core/src/main/java/org/elasticsearch/index/indexing/IndexingStats.java

```diff
@@ -155,6 +161,7 @@ public void readFrom(StreamInput in) throws IOException {
             indexCount = in.readVLong();
             indexTimeInMillis = in.readVLong();
             indexCurrent = in.readVLong();
+            indexFailedCount = in.readVLong();
```

Comment:
Yup - just that read and write you inserted. Without it this isn't compatible with 2.0 servers. Usually people only run mixed version clusters during a rolling upgrade but its nice if everything continues to work then.